src/vs/workbench/contrib/search/browser/searchActions.ts

```diff
@@ -465,7 +483,9 @@ export class RemoveAction extends AbstractSearchAndReplaceAction {
 			this.viewer.setSelection([nextFocusElement], getSelectionKeyboardEvent());
 		}
 
-		this.element.parent().remove(<any>this.element);
+		elementsToRemove.forEach((currentElement, index) =>
+			currentElement?.parent().remove(<any>currentElement)
```

Comment:
`?` needed? And this cast is unfortunate but I guess it was there before. Is it still needed?

Reply:
seems that `remove` takes `FileMatch | FolderMatch | (FileMatch | FolderMatch)[]` but `RenderableMatch` can't be statically casted to that. That's probably why it casts to `any`.